Packs/Clarizen/Integrations/ClarizenIAM/ClarizenIAM.py

```diff
@@ -54,7 +54,7 @@ def test(self):
         return self._http_request(method='POST', url_suffix=uri, data={}, params=params)
 
     def get_user_by_id(self, user_id):
-        uri = f'/data/objects/{user_id}'
+        uri = f'/data/objects{user_id}'
```

Comment:
shouldn't it be like this?
```suggestion
        uri = f'/data/objects/{user_id}'
```